pontoon/base/migrations/0098_move_ms_locales_to_db.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-04-13 21:01
+from __future__ import unicode_literals
+
+from bulk_update.helper import bulk_update
+from django.db import migrations, models
+
+
+TRANSLATOR_LOCALES = {
+    u'ar': u'ar',
+    u'bg': u'bg',
+    u'bs-Latn': u'bs',
+    u'ca': u'ca',
+    u'cs': u'cs',
+    u'cy': u'cy',
+    u'da': u'da',
+    u'de': u'de',
+    u'el': u'el',
+    u'en': u'en-GB',
+    u'es': u'es',
+    u'et': u'et',
+    u'fa': u'fa',
+    u'fi': u'fi',
+    u'fr': u'fr',
+    u'he': u'he',
+    u'hi': u'hi-IN',
+    u'hr': u'hr',
+    u'ht': u'ht',
+    u'hu': u'hu',
+    u'id': u'id',
+    u'it': u'it',
+    u'ja': u'ja',
+    u'ko': u'ko',
+    u'lt': u'lt',
+    u'lv': u'lv',
+    u'ms': u'ms',
+    u'mt': u'',
+    u'mww': u'',
+    u'nl': u'nl',
+    u'no': u'',
+    u'otq': u'',
+    u'pl': u'pl',
+    u'pt': u'pt-PT',
+    u'ro': u'ro',
+    u'ru': u'ru',
+    u'sk': u'sk',
+    u'sl': u'sl',
+    u'sr-Cyrl': u'sr',
+    u'sv': u'sv-SE',
+    u'th': u'th',
+    u'tlh': u'',
+    u'tlh-Qaak': u'',
+    u'tr': u'tr',
+    u'uk': u'uk',
+    u'ur': u'ur',
+    u'vi': u'vi',
+    u'yua': u'',
+    u'zh-CHS': u'zh-CN',
+}
+
+TERMINOLOGY_LOCALES = {
+    u'af-za': u'af',
+    u'am-et': u'am',
+    u'ar-sa': u'ar',
+    u'as-in': u'as',
+    u'be-by': u'be',
+    u'bg-bg': u'bg',
+    u'ca-es': u'ca',
+    u'cs-cz': u'cs',
+    u'cy-gb': u'cy',
+    u'da-dk': u'da',
+    u'de-de': u'de',
+    u'el-gr': u'el',
+    u'et-ee': u'et',
+    u'eu-es': u'eu',
+    u'fa-ir': u'fa',
+    u'fi-fi': u'fi',
+    u'fr-fr': u'fr',
+    u'ga-ie': u'ga-IE',
+    u'gl-es': u'gl',
+    u'gu-in': u'gu-IN',
+    u'ha-Latn-ng': u'ha',
+    u'he-il': u'he',
+    u'hi-in': u'hi-IN',
+    u'hr-hr': u'hr',
+    u'hu-hu': u'hu',
+    u'hy-am': u'hy-AM',
+    u'id-id': u'id',
+    u'ig-ng': u'ig',
+    u'is-is': u'is',
+    u'it-it': u'it',
+    u'ja-jp': u'ja',
+    u'ka-ge': u'ka',
+    u'kk-kz': u'kk',
+    u'km-kh': u'km',
+    u'kn-in': u'kn',
+    u'ko-kr': u'ko',
+    u'lo-la': u'lo',
+    u'lt-lt': u'lt',
+    u'lv-lv': u'lv',
+    u'ml-in': u'ml',
+    u'mr-in': u'mr',
+    u'ne-np': u'ne-NP',
+    u'nl-nl': u'nl',
+    u'pa-in': u'pa-IN',
+    u'pl-pl': u'pl',
+    u'ro-ro': u'ro',
+    u'ru-ru': u'ru',
+    u'si-lk': u'si',
+    u'sk-sk': u'sk',
+    u'sl-si': u'sl',
+    u'sq-al': u'sq',
+    u'sv-se': u'sv-SE',
+    u'te-in': u'te',
+    u'th-th': u'th',
+    u'tr-tr': u'tr',
+    u'uk-ua': u'uk',
+    u'ur-pk': u'ur',
+    u'vi-vn': u'vi',
+    u'wo-sn': u'wo',
+    u'yo-ng': u'yo',
+
+    # Batch of locales that require a review.
+    u'az-Latn-az': u'az',
```

Comment:
It was lower case in settings: `az-latn-az`. That code is bullshit but for some reason microsoft uses this version and it is unique for microsoft.